pyriemann_qiskit/classification.py

```diff
@@ -273,18 +268,20 @@ class QuanticSVM(QuanticClassifierBase):
 
     """
 
-    def __init__(self, gamma=None, **parameters):
+    def __init__(self, gamma='scale', **parameters):
         QuanticClassifierBase.__init__(self, **parameters)
         self.gamma = gamma
 
     def _init_algo(self, n_features):
-        # Although we do not train the classifier at this location
-        # training_input are required by Qiskit library.
         self._log("SVM initiating algorithm")
         if self.quantum:
-            classifier = QSVM(self._feature_map, self._training_input)
+            quantum_kernel = \
+                QuantumKernel(feature_map=self._feature_map,
+                              quantum_instance=self._quantum_instance)
+            classifier = QSVC(quantum_kernel=quantum_kernel,
+                              gamma=self.gamma)
         else:
-            classifier = SklearnSVM(self._training_input, gamma=self.gamma)
+            classifier = SVC(gamma=self.gamma)
```

Comment:
Now that we use directly SVC, we can probably pass more than just `gamma` as an hyperparameter